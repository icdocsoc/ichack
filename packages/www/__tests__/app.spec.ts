import { expect, test } from 'vitest';
import { mountSuspended } from '@nuxt/test-utils/runtime';
import Page from '../app.vue';

test('Match snapshot', async () => {
  const component = await mountSuspended(Page);
  expect(component.text()).toMatchSnapshot();
});
