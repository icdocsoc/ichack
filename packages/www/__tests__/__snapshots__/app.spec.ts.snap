// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Match snapshot 1`] = `"ICHACK '25 Landing Page"`;
